hnn_core/tests/test_dipole.py

```diff
@@ -48,6 +48,12 @@ def test_dipole(tmpdir, run_hnn_core_fixture):
                        "average of 2 trials"):
         dipole_avg = average_dipoles([dipole_avg, dipole_read])
 
+    # average single dipole object
+    dpl_avg1 = average_dipoles([dipole])
```

Comment:
What about this for a test. You can create a `Dipole` object with two simulations, but second the data stored under the second trials to `dpl[1].data = np.zeroes(len(dpl[0].times))`.

Then your test can be 
```py
assert np.all(dpl_avg == dpl[0].data / 2)
```
I think the current test is good and should remain, but it'd be good to make sure the average function is actually doing some averaging too.

Reply:
@ntolley Sounds good! I'll give it a try.

Reply:
@ntolley I tried your implementation of setting the data for the second trial to zero. However the assertion
> `assert np.all(dpl_avg == dpl[0].data / 2)`

causes the entire array to be set to 0. floats. This raises an IndexError later when trying to compute the average because numpy isn't able to compute a mean for:
> `[0. 0. 0. ... 0. 0. 0.]`

I'll try and find a way around this in the meantime. I might have to rely on a different way to set the data to 0.

Reply:
you should use `np.allclose` to compare arrays